builder/actions.py

```diff
@@ -88,6 +88,7 @@ def save(*, draft):
     All this does is unset the draft_owner attribute.
     """
 
+    assert not draft.code_objs.filter(status__in=["?", "!"]).exists()
```

Comment:
A comment with this line just to sum up what it's asserting would be great, I can't remember what `?` and `!` mean.